css-paint-api/Overview.bs

```diff
@@ -605,7 +614,7 @@ When the user agent wants to <dfn>invoke a paint callback</dfn> given |name|, |i
     8. Let |renderingContext| be the result of <a>create a PaintRenderingContext2D object</a> given:
```

Comment:
has the "create a PaintRenderingContext2D object" algorithm been updated to receive a contextAttributes object?

Reply:
Good catch, I have updated that section.